zerver/actions/uploads.py

```diff
@@ -43,8 +43,8 @@ def do_claim_attachments(
         if not validate_attachment_request(user_profile, path_id):
             # Technically, there are 2 cases here:
             # * The user put something in their message that has the form
-            # of an upload, but doesn't correspond to a file that doesn't
-            # exist.  validate_attachment_request will return None.
+            # of an upload, but corresponds to a file that doesn't exist.
+            # validate_attachment_request will return None.
```

Comment:
I think more correct would be "of an upload URL, but does not actually correspond to a previously uploaded file".

"file that doesn't exist" is awkward phrasing.

Reply:
Done as separate commit. Would you prefer them being squashed together?

Reply:
Yes, please squash them. Thank you!